src/ui/js/Profile.js

```diff
@@ -51,10 +51,16 @@ Profile.showPage = function() {
 Profile.buildProfileTable = function() {
   var table = $('<table>', { 'class': 'profileTable', });
 
+  var profileName=$('<spane>');
```

Comment:
Hmm, what is `<spane>`?  Is that an HTML element?  Once i bring the site up, i'll see if firebug recognizes it, but i suspect it's a typo for `<span>`.